monkey/infection_monkey/network/test_postgresql_fingerprint.py

```diff
@@ -0,0 +1,86 @@
+from unittest import TestCase
+from unittest.mock import Mock
+
+from infection_monkey.network.postgresql_fingerprint import PostgreSQLFinger
+
+IRRELEVANT_EXCEPTION_STRING = "This is an irrelevant exception string."
+
+RELEVANT_EXCEPTION_STRINGS =\
+    {
+        'pwd_auth_failed': 'FATAL:  password authentication failed for user "root"',
```

Comment:
Wonderful, now we know what kind of exceptions we can expect from postgres in one quick look